### DISCUSSION TITLE
Windows 10 does not see .quasar.env.json file

### DISCUSSION THREAD

**Discussion by ayazici-vekend** [quasarframework/quasar#9972] [CLI - Electron mode]
I was working on linux before and my project was working fine, and it still does. I switched to Windows10 later on, but now Windows does not see my env file and when i run the app with the command **"yarn dev"** it gives the error:

_yarn run v1.22.10
$ QENV=development quasar dev
'QENV' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command._

.quasar.env.json file:
`
{
  "development": {
    "ENV_TYPE": "Running Development",
    "ENV_DEV": "Development"
  },
  "production": {
    "ENV_TYPE": "Running Production",
    "ENV_PROD": "Production"
  },
  "test": {
    "ENV_TYPE": "Running Test",
    "ENV_Test": "Test"
  }
}
`

package.json file:
`
{
  "scripts": {
    "dev": "QENV=development quasar dev"
  },
  "dependencies": {
    "@quasar/extras": "^1.0.0",
    "quasar": "^1.0.0"
  },
  "devDependencies": {
    "@quasar/app": "^2.0.0",
    "@quasar/quasar-app-extension-qenv": "^1.0.3"
  },
  "engines": {
    "node": ">= 10.18.1",
    "npm": ">= 6.13.4",
    "yarn": ">= 1.21.1"
  }
}
`

**REMINDER:** Works perfectly on linux, but gives the upper error in windows 10

**Comment by justinbeaty** [comment]
@ayazici-vekend Try changing your package.json look like:

```
{ "scripts": { "dev": "set QENV=development && quasar dev" }, ... 
```

Source: https://stackoverflow.com/a/40806995

**Reply by ayazici-vekend** [reply]
@justinbeaty  thank you so much you saved me, its perfect now

**Reply by ayazici-vekend** [reply]
@justinbeaty now i have a different problem. It says **! App Extension (qenv): Missing 'development ' from .quasar.env.json; skipping** while compiling, and since i have some variables inside that development they are not seen.

**Reply by justinbeaty** [reply]
@ayazici-vekend very strange, I'm really not a Windows guy at all, but something that sticks out to me is that it is trying to load the key `'development '` (notice the space.)

Maybe try one of these:

```
"dev": "set QENV=\"development\" && quasar dev"

"dev": "set \"QENV=development\" && quasar dev"

"dev": "(set QENV=development) && quasar dev"
```

Maybe one will work?

**Reply by ayazici-vekend** [reply]
@justinbeaty yes exactly, it tries to read the key with space at the end since i put space in between 'development' and '&&'.
Thank you very much.
